templates/components/common/body.html

```diff
@@ -4,4 +4,5 @@
         {{#block "page"}} {{/block}}
     </div>
     {{> components/common/modal}}
+    {{> components/common/alert-modal}}
```

Comment:
I guess we still need the regular modal too?

Reply:
Yeah, the regular modal is used for things like the product quick view. Alert modal is more specialized.